x-pack/plugins/apm/common/apm_feature_flags.ts

```diff
@@ -47,6 +49,14 @@ const apmFeatureFlagMap = {
     default: true,
     type: t.boolean,
   },
+  [ApmFeatureFlagName.SchemaAvailable]: {
+    default: true,
+    type: t.boolean,
+  },
+  [ApmFeatureFlagName.StorageExplorerAvailable]: {
+    default: false,
```

Comment:
Shouldn't this be `true` as default as the other flags?

Reply:
Left over from testing